src/Compilers/Core/Portable/MetadataReader/PEModule.cs

```diff
@@ -1841,7 +1870,7 @@ private static bool CrackStringAndIntInAttributeValue(out StringAndInt value, re
                 CrackIntInAttributeValue(out value.IntValue, ref sig);
         }
 
-        internal static bool CrackStringInAttributeValue(out string value, ref BlobReader sig)
+        internal static bool CrackStringInAttributeValue(out string? value, ref BlobReader sig)
```

Comment:
I would assume the return value accurately reflects the null status of the string, or is it possible to return true and be null? If its accurate consider using `[NotNullWhen(true)]` as above